testing/test_testcase1.py

```diff
@@ -277,6 +278,33 @@ def test_return(self):
         # Check return is valid for time period
         self.assertTrue(res['time_period'] is None)
 
+class ComputationalTimeRatio(unittest.TestCase):
+    '''Test the computational time ratio KPI explicitly.
+
+    '''
+
+    def setUp(self):
+        '''Setup for each test.
+
+        '''
+
+        self.url = 'http://127.0.0.1:5000'
+
+    def test_kpi(self):
+        '''Tests the calculation of the kpi.
+
+        '''
+
+        # Run test
+        requests.put('{0}/initialize'.format(self.url), data={'start_time':0, 'warmup_period':0})
+        step = requests.get('{0}/step'.format(self.url)).json()
+        for i in range(5):
```

Comment:
Should we test that we can also handle variable control steps for computational time ratio? I mean, we could set a variable step as function of the `i` index either in this same test or [here](https://github.com/ibpsa/project1-boptest/blob/bf9d49873ec3b475c9324092141628b32ea9c785/testing/test_kpis.py#L131).

Reply:
Yes, good point, thanks.  I can implement.

Reply:
Added in https://github.com/ibpsa/project1-boptest/pull/329/commits/d470b43388002cb96568d2f416154736f60e8a25.

Reply:
Thanks @dhblum! looking good.